connectors/services/job_scheduling.py

```diff
@@ -167,29 +170,40 @@ async def _run(self):
                 await self.sync_job_index.close()
         return 0
 
-    async def _scheduled_sync(self, connector, job_type):
+    async def _try_schedule_sync(self, connector, job_type):
+        last_wake_up_time = self.last_wake_up_time
+        this_wake_up_time = datetime.utcnow()
+
+        print(
+            f"Last time woke up at {last_wake_up_time}, woke up at {this_wake_up_time}"
+        )
```

Comment:
```suggestion
```

I guess this was used for local debugging?

Reply:
Yup, need to change it to logger.debug!